packages/performance/src/services/remote_config_service.test.ts

```diff
@@ -88,15 +94,18 @@ describe('Performance Monitoring > remote_config_service', () => {
     it('gets the config from the local storage if available and valid', async () => {
       // After global clock. Config not expired.
       const EXPIRY_LOCAL_STORAGE_VALUE = '1556524895330';
-      storageGetItemStub.callsFake(
-        storageGetItemFakeFactory(
-          EXPIRY_LOCAL_STORAGE_VALUE,
-          STRINGIFIED_CONFIG
+      const getItemStub = stub();
+      storageStub.value({
+        getItem: getItemStub.callsFake(
+          storageGetItemFakeFactory(
+            EXPIRY_LOCAL_STORAGE_VALUE,
+            STRINGIFIED_CONFIG
+          )
```

Comment:
can we put the stubbing part in a function which returns the stub? Something like the pattern discussed here: go/sifers.

Reply:
Done. PTAL.